backend/routes/adminRouter.js

```diff
@@ -0,0 +1,47 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const mongoose = require('mongoose');
+
+const Students = require('../models/students');
+
+const dishRouter = express.Router();
```

Comment:
What is the meaning of dishRouter.

Reply:
It should be adminRouter, nor dishRouter

Reply:
Yes it should be adminRouter